sandbox/tests/test scenes/primitive mesh/01 - sphere.appleseed

```diff
@@ -0,0 +1,180 @@
+                <parameter name="enable_diagnostics" value="false" />
+                <parameter name="max_samples" value="9" />
+                <parameter name="min_samples" value="16" />
+                <parameter name="quality" value="2.000000" />
+            </parameters>
+            <parameters name="drt">
+                <parameter name="dl_light_samples" value="1.000000" />
+                <parameter name="enable_ibl" value="true" />
+                <parameter name="ibl_env_samples" value="1.000000" />
+                <parameter name="max_path_length" value="0" />
+                <parameter name="rr_min_path_length" value="3" />
+            </parameters>
+            <parameters name="generic_frame_renderer">
+                <parameter name="passes" value="1" />
+            </parameters>
+            <parameters name="generic_tile_renderer">
+            </parameters>
+            <parameters name="pt">
+                <parameter name="dl_light_samples" value="1.000000" />
+                <parameter name="enable_caustics" value="false" />
+                <parameter name="enable_dl" value="true" />
+                <parameter name="enable_ibl" value="true" />
+                <parameter name="ibl_env_samples" value="1.000000" />
+                <parameter name="max_path_length" value="0" />
+                <parameter name="next_event_estimation" value="true" />
+                <parameter name="rr_min_path_length" value="3" />
+            </parameters>
+            <parameters name="sppm">
+                <parameter name="alpha" value="0.700000" />
+                <parameter name="dl_mode" value="sppm" />
+                <parameter name="enable_caustics" value="true" />
+                <parameter name="enable_ibl" value="true" />
+                <parameter name="env_photons_per_pass" value="100000" />
+                <parameter name="initial_radius" value="1.000000" />
+                <parameter name="light_photons_per_pass" value="100000" />
+                <parameter name="max_photons_per_estimate" value="100" />
+                <parameter name="path_tracing_max_path_length" value="0" />
+                <parameter name="path_tracing_rr_min_path_length" value="3" />
+                <parameter name="photon_tracing_max_path_length" value="0" />
+                <parameter name="photon_tracing_rr_min_path_length" value="3" />
+                <parameter name="photon_type" value="mono" />
+            </parameters>
+            <parameters name="uniform_pixel_renderer">
+                <parameter name="decorrelate_pixels" value="true" />
+                <parameter name="force_antialiasing" value="false" />
+                <parameter name="samples" value="16" />
+            </parameters>
+        </configuration>
+        <configuration name="interactive" base="base_interactive">
+            <parameter name="lighting_engine" value="pt" />
+            <parameter name="rendering_threads" value="1" />
```

Comment:
It's best not to force a specific number of rendering threads in test scenes. You could either remove this line or remove the override in the Interactive Mode rendering settings in appleseed.studio.